syntax = "proto3";

package toll_roads;

service TollRoadsService {
    rpc TollRoads (TollRoadsRequest) returns (TollRoadsResponse);
}

message TollRoadsRequest {
    string display_name = 1;
}

message TollRoadsResponse {
    float bonus_amount = 1;
}
